pkg/virt-operator/install-strategy/create.go

```diff
@@ -1827,6 +1900,12 @@ func SyncAll(kv *v1.KubeVirt,
 		return false, err
 	}
 
+	// create/update PrometheusRules
```

Comment:
We will have to remove no longer existing objects after `// -------- CLEAN UP OLD UNUSED OBJECTS --------`. Otherwise we will leave old objects in place.

Reply:
Fixed.